pyscriptjs/src/pyscript.py

```diff
@@ -368,15 +368,17 @@ def child_appended(self, child):
 
 
 class OutputCtxManager:
-    def __init__(self, out=None, output_to_console=True, append=True):
+    def __init__(self, out=None, err=None, output_to_console=True, append=True):
```

Comment:
Not having `err` here was intentional because the separation of `err` vs `out` happens in `OutputManager`. I think adding `err` here adds ambiguity to the API because `out` and `err` are not clearly separate.  

I understand the intention here (add semantic value to the input names) but I think the final result is kind of confusing.

Reply:
I added this because
``` python
sys.stderr = self._err_manager = OutputCtxManager(
    err=err, output_to_console=output_to_console, append=append
)
```
inits an `OutputCtxManager` as `_err_manager` with `err` as an argument, but `OutputCtxManager` had no parameter for `err`.

Would that still work even without the parameter in the __init__?

Reply:
This could be:

```
sys.stderr = self._err_manager = OutputCtxManager(
    out=err, output_to_console=output_to_console, append=append
)
```

right?

Reply:
That's true, looks a bit odd but should work. Meaning we could completely remove err from OutputCtxManager, as it's not used in any way.

Reply:
We can try to find a better naming to make it less weird after the PR was merged. :) 

This is an internal API and not that exposed to users

Reply:
That's a good idea. I added the last commit, which removes the err parameter in `OutputCtxManager`.
As you said, we should find a better name for `OutputCtxManager`.

From my side that should be everything, feel free to review and resolve if it fits. :)